src/osbuild-manifests/platform.live-iso.ipp.yaml

```
@@ -0,0 +1,73 @@
+# This file defines the pipeline for building the live ISO.
+version: '2'
+pipelines:
+  - name: live-iso
+    stages:
+      - type: org.osbuild.copy
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              - name:metal
+        options:
+          paths:
+            - from:
+                mpp-format-string: "input://tree/{artifact_name_prefix}-metal.{arch}.raw"
+              to: tree:///metal.raw
+      - type: org.osbuild.copy
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              - name:metal4k
+        options:
+          paths:
+            - from:
+                mpp-format-string: "input://tree/{artifact_name_prefix}-metal4k.{arch}.raw"
+              to: tree:///metal4k.raw
+      # We need to be able to create efiboot.img, a FAT filesystem image. It's
+      # hard to setup loopback devices from within the stage, so just do it
+      # here. This should normally be conditional on the architecture but it
+      # doesn't hurt either since it's a tiny file.
+      - type: org.osbuild.truncate
+        options:
+          filename: efiboot.img
+          size:
+            mpp-format-string: '{live_efiboot_img_size_mb * 1024 * 1024}' 
+      - type: org.osbuild.coreos.live-iso
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              # XXX: we should rename this pipeline now that we don't use it as
+              # a buildroot
+              - name:build
```

if https://github.com/coreos/coreos-assembler/pull/3946 merges we'll need to update this to use the new name.

```suggestion
              # XXX: we should rename this pipeline now that we don't use it as
              # a buildroot
              - name:build
```